public/app/features/dashboard/components/DashNav/DashNav.tsx

```diff
@@ -380,7 +380,9 @@ export const DashNav = React.memo<Props>((props) => {
     addCustomContent(customRightActions, buttons);
 
     buttons.push(renderTimeControls());
-    buttons.push(tvButton);
+    if (!config.isPublicDashboardView) {
```

Comment:
couldn't find another way of not showing tvButton. Had to use this flag

Reply:
Why can't we show it ? Seem harmless

Reply:
I tested it yesterday and it wasn't working properly for pubdash. I was going to create a new ticket to add it.